webpack/ForemanInventoryUpload/Components/PageHeader/components/CloudConnectorButton/CloudConnectorButton.js

```diff
@@ -10,26 +10,26 @@ export const CloudConnectorButton = ({ status, onClick, jobLink }) => {
       <Popover
         bodyContent={
           <div>
-            {__('Cloud connector job is still running, you can view it here:')}
+            {__('Cloud connector setup is in progress now: ')}
             <a href={jobLink} target="_blank" rel="noopener noreferrer">
-              {__('Open job')}
+              {__('Cloud connector job link')}
```

Comment:
How about:

`Cloud connector setup has started: view the job in progress`

![Screenshot (39)](https://user-images.githubusercontent.com/26363699/106872815-68adf000-66dc-11eb-8bab-a99e0c71eefc.png)